src/guiFormSpecMenu.cpp

```diff
@@ -2478,11 +2483,20 @@ void GUIFormSpecMenu::drawMenu()
 	video::IVideoDriver* driver = Environment->getVideoDriver();
 
 	v2u32 screenSize = driver->getScreenSize();
-	core::rect<s32> allbg(0, 0, screenSize.X ,	screenSize.Y);
-	if (m_bgfullscreen)
+	core::rect<s32> allbg(0, 0, screenSize.X, screenSize.Y);
+
+	if (m_bgfullscreen) {
+		v3f formspec_bgcolor = g_settings->getV3F("formspec_fullscreen_bg_color");
+		m_bgcolor = video::SColor(
```

Comment:
i will change this to prepare bgcolor in constructor permitting to have defaults + override